packages/fluent/scss/listbox/_variables.scss

```diff
@@ -29,13 +29,13 @@ $kendo-listbox-line-height: var( --kendo-line-height, normal ) !default;
 
 /// Background color of the listbox component.
 /// @group listbox
-$kendo-listbox-bg: $kendo-component-bg !default;
+$kendo-listbox-bg: var( --kendo-component-bg, #{$kendo-component-bg} ) !default;
```

Comment:
I think it's better to give a different fallback to `$kendo-listbox-bg`, `-text` and `-border` here as the vars already have the same fallbacks.

Like so $kendo-listbox-bg: var( --kendo-component-bg, transparent ) or initial.

This will ensure there is a fallback if `$kendo-component-bg` is set to null for example.